.travis.yml

```diff
@@ -0,0 +1,72 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+
+matrix:
+  include:
+    - env: USE_ADIOS=OFF
+  # - env: USE_ADIOS=SERIAL
```

Comment:
disabled, waiting for https://github.com/travis-ci/apt-package-whitelist/issues/2187